doc/release/upcoming_changes/15271.compatibility.rst

```diff
@@ -0,0 +1,31 @@
+Error type changes in universal functions
+-----------------------------------------
+The universal functions may now raise different errors
+on invalid input in some cases.
+The main changes should be that a ``RuntimeError`` was
+replaced with a more fitting ``TypeError``.
+When multiple errors were present in the same call,
+NumPy may now raise a different one.
+
+
+``__array_ufunc__`` argument validation
+---------------------------------------
+NumPy will now partially validate arguments before calling
+``__array_ufunc__``.  Previously, it was possible to pass
+on invalid arguments (such as a non-existing keyword
+argument) when dispatch was known to occur.
+
+
+``__array_ufunc__`` and additional positional arguments
+-------------------------------------------------------
+Previously, all positionally passed arguments were check for
```

Comment:
Sorry, missed a small mistake: check -> checked

Reply:
Thanks.  Yeah I have to remember to add `[skip CI]` more often.  Seems it didn't pan out, because I pushed before CI was finished I guess.